### PR TITLE
Fix UT 'logs will not be displayed' error

### PR DESCRIPTION
**Pull request by TaoZou1**
The UT TestSRGetClient will report error "
log.SetLogger(...) was never called; logs will not be displayed.". 
Add log initialization

### CODE REVIEW

**Review by zhengxiexie — APPROVED** [review]

File: pkg/config/config.go [hunk 1]
```diff
@@ -358,7 +358,7 @@ func (nsxConfig *NsxConfig) validateCert() error {
 		return err
 	}
 	if caCount > 0 {
-		configLog.Infof("validate CA file: %s", caCount)
+		configLog.Infof("validate CA file: %d", caCount)
```
Comment (wenqiq) [inline pkg/config/config.go:361]: Capitalize the log sentence?

Comment (TaoZou1) [reply pkg/config/config.go]: Done

**Review by wenqiq — APPROVED** [review]
lgtm

**Review by timdengyun — APPROVED** [review]
LGTM

**Pull request merged**